src/main/java/hudson/plugins/cobertura/targets/CoverageResult.java

```diff
@@ -1,14 +1,7 @@
 package hudson.plugins.cobertura.targets;
 
-import hudson.model.AbstractBuild;
-import hudson.model.Api;
-import hudson.model.Item;
-import hudson.model.Run;
-import hudson.plugins.cobertura.BuildUtils;
-import hudson.plugins.cobertura.Chartable;
-import hudson.plugins.cobertura.CoberturaBuildAction;
-import hudson.plugins.cobertura.CoverageChart;
-import hudson.plugins.cobertura.Ratio;
+import hudson.model.*;
+import hudson.plugins.cobertura.*;
```

Comment:
It is generally preferable to avoid it in the production code, because there is a risk of ambiguity if one of the packages introduces a competing class.